projects/angular/src/utils/i18n/common-strings.default.ts

```diff
@@ -68,10 +68,10 @@ export const commonStringsDefault: ClrCommonStrings = {
   // Stack View
   stackViewChanged: 'Value changed.',
   // Responsive Nav
-  responsiveNavToggleOpen: 'Open navigation menu',
-  responsiveNavToggleClose: 'Close navigation menu',
-  responsiveNavOverflowOpen: 'Open navigation overflow menu',
-  responsiveNavOverflowClose: 'Close navigation overflow menu',
+  responsiveNavToggleOpen: 'Navigation menu',
```

Comment:
still kept the `*ToggleOpen/Close` commonstrings so that apps could go wild with customization.